### PR TITLE
Patch crun

### PR DESCRIPTION
**Pull request by fwilhe**
Workaround as suggested in https://github.com/actions/runner-images/issues/9425

### CODE REVIEW

File: .github/actions/setup/action.yml [hunk 1]
```diff
@@ -1,6 +1,14 @@
 runs:
   using: composite
   steps:
+
+    # Workaround for https://github.com/actions/runner-images/issues/9425, to be removed once the issue is resolved
+    - name: patch crun (Workaround for https://github.com/actions/runner-images/issues/9425)
+      shell: bash
+      run: |
+        curl -Lo ./crun https://github.com/containers/crun/releases/download/1.14.3/crun-1.14.3-linux-amd64
+        sudo install crun /usr/bin/crun
```
Comment (Vincinator) [inline .github/actions/setup/action.yml:10]: I suggest to verify signature of the downloaded crun binary
https://github.com/containers/crun/releases/download/1.14.3/crun-1.14.3-linux-amd64.asc

Comment (fwilhe) [reply .github/actions/setup/action.yml]: Good suggestion, thanks. I had issues with doing it with gpg for reasons, probably I don't understand the gpg cli good enough. Resorted to using sha sum for now, sadly they don't seem to publish it for releases. I think this should be safe enough for our purposes now, but I'm also open to other suggestions.

Comment (fwilhe) [reply .github/actions/setup/action.yml]: So the 'good' sha sum in here was created by me because sadly the github release does not have sha sums

Comment (Vincinator) [reply .github/actions/setup/action.yml]: Yepp, that works for me

**Review by Vincinator — APPROVED** [review]

**Pull request merged**